src/io/files.js

```diff
@@ -421,6 +421,13 @@ p5.prototype.loadTable = function(path) {
           sep = ',';
           separatorSet = true;
         }
+      } else if (arguments[i] === 'ssv') {
+        if (separatorSet) {
+          throw new Error('Cannot set multiple separator types.');
```

Comment:
let's remove these errors throughout, while we're here.
incorrect argument orders should be caught by the friendly error system, not checked within this logic
would you be up for making this change to remove each of the errors here?

Reply:
I looked into it but i dont feel confident enough to implement the FES checks.
However I added the new options in the documentation.

Reply:
ok no problem. thanks for making the other edit